Tests 11, 17

giuliorm opened the pull request:
Tests

giuliorm committed:
tests

giuliorm committed:
Merge branch 'master' of http://github.com/bmkodoo/javacourses2016 into tests

giuliorm committed:
tests disabled

giuliorm committed:
tests disabled-2

giuliorm committed:
test 17

giuliorm committed:
test disablings added

giuliorm committed:
test 17

nkarulin commented on src/test/java/com/epam/javacourses2016/Task17/SolverTask17Test.java:
```diff
@@ -1,69 +0,0 @@
-import java.util.HashSet;
-import java.util.Set;
-
-public class SolverTask17Test {
```
За что ты удалила уже имеющийся тест? Чем больше тестов, тем лучше

giuliorm replied:
Мой файл с тестом назывался так же, я его переименовала, но не помню, чтобы удаляла существующий. что теперь с этим делать, скачать заново? он у меня в ветке с тестами даже лежит. просто файлы конфликтовали...

giuliorm replied:
К  тому же, данные в "имеющемся" тесте заданию не соответствуют, например, кейс:

> new ArrayList<double[][]>() {
>                     {
>                         add(new double[][]{{1, 1}, {3, 3}});
>                         add(new double[][]{{3, 1}, {1, 3}});
>                         add(new double[][]{{1, 4}, {3, 6}});
>                         add(new double[][]{{1, 6}, {3, 4}});
>                     }
>                 }, new double[][]{{2, 2,}, {2, 5}}}
>         };
> 

Требуется по заданию из _существующего списка_ выбрать точки с минимальными абсциссами, когда как точки (2,2) и (2,5) вообще в список не входят

giuliorm committed:
tests 11 and 17 renamings

giuliorm committed:
Merge with branch master and conflict resolvings

giuliorm committed:
test 17 conflicts resolving

giuliorm committed:
Merge remote-tracking branch 'origin/tests' into tests

giuliorm committed:
test 17 conflicts resolving

bmkodoo commented:
Shippable found some fails

giuliorm committed:
Test 2 changed: logic of these tests was uncompatible, cause first test required the name of the source directory to be in the list, and the second did not
Test 11 changed: in the array of 8 the 8th element must be the result and in the array of five the 2nd element must be the result ( 1 2 3 4 5 6 7 8 -> 1- 2 3- 4 5- 6 7- 8 - the next "first" element is 2 (7, then 8,  then 2), 2 4 6 8 -> 2- 4 6- 8 (6 the last then 8, then 4 because 2 is already gone) -> 4 8 -> 4- and then we have 8 left. The same 1 2 3 4 5 -> 1- 2 3- 4 5- next we have 5, then 2, then 4 -> 4 2 -> 4- 2 and we have 2 left).

+

giuliorm committed:
Merge branch 'master' of http://github.com/bmkodoo/javacourses2016 into tests

# Conflicts:
#	src/test/java/com/epam/javacourses2016/task11/SolverTask11Test.java

giuliorm commented:
Поменяй, пожалуйста, в мастере директорию с тестами Task17 на task17 (маленькая буква), у меня от этого не собирается основная ветка с заданиями

/root/src/github.com/bmkodoo/javacourses2016/src/test/java/com/epam/javacourses2016/Task17/Task17Test.java:29: error: analyze(Set<Segment>) is not public in SolverTask17; cannot be accessed from outside package

bmkodoo commented:
Thanks!
